chainer/links/connection/lstm.py

```diff
@@ -91,6 +91,14 @@ class LSTM(LSTMBase):
     It also maintains *states*, including the cell state and the output
     at the previous time step. Therefore, it can be used as a *stateful LSTM*.
 
+    This link supports variable length inputs. The mini-batch size of ``c``
```

Comment:
ditto (add the following sentence: The mini-batch size of the current input must be equal to or smaller than that of the previous one.)